content/en/blog/_posts/2022-10-18-kubernetes-1.26-deprecations-and-removals.md

```diff
@@ -0,0 +1,116 @@
+
+The `userspace` proxy mode, deprecated for over a year, is [no longer supported on either Linux or Windows](https://github.com/kubernetes/kubernetes/pull/112133) and will be removed in this release. Users should use `iptables` or `ipvs` on Linux, or `kernelspace` on Windows: using `--mode userspace` will now fail.
+
+### Removal of in-tree OpenStack cloud provider
+
+The in-tree cloud provider for OpenStack (and the Cinder volume provider) [will be removed](https://github.com/kubernetes/kubernetes/pull/67782). Users are advised to use the external cloud provider and CSI driver from https://github.com/kubernetes/cloud-provider-openstack instead.
+
+### Deprecation of non-inclusive `kubectl` flag
+
+A part of the implementation effort of the [Inclusive Naming Initiative](https://www.cncf.io/announcements/2021/10/13/inclusive-naming-initiative-announces-new-community-resources-for-a-more-inclusive-future/), the `--prune-whitelist` flag [will be deprecated](https://github.com/kubernetes/kubernetes/pull/113116), and replaced with `--prune-allowlist`. Users that use this flag are strongly advised to make the necessary changes prior to the final removal of the flag, in a future release.
+
+### Deprecations of `kube-apiserver` flag
+
+The `master-service-namespace` flag was [already deprecated](https://github.com/kubernetes/kubernetes/pull/112797) and doesn't have any effect, so we do not expect any impact from this deprecation. It will nonetheless be explicitly marked as deprecated in v1.26, preparing the removal in a future release.
+
+### Deprecation of several unused `kubectl run` flags
+
+Several unused flags will be [marked as deprecated](https://github.com/kubernetes/kubernetes/pull/112261), including:
+
+* `--cascade`
+* `--filename`
+* `--force`
+* `--grace-period`
+* `--kustomize`
+* `--recursive`
+* `--timeout`
+* `--wait`
+
+These flags are already ignored so no impact is expected: the explicit deprecation sets a warning message and prepares the removal of the flags in a future release.
+
+### Removal of deprecated klog flags
+
+This change completes the [deprecations of `klogs` flags](https://github.com/kubernetes/kubernetes/pull/112120) which are no longer supported, and have been removed from Kubernetes components.
+
+
+## Additional changes for Kubernetes v1.26 {#additional-changes}
+
+On top of the described removals, this release will also include some additional changes to take into consideration: while not deprecations or removals, they introduce new features (or a new default behavior) through the graduation to Stable in this release; some of those changes are highlighted here.
+
+### Support for Windows privileged containers
+
+Privileged container support allows containers to run with similar access to the host as processes that run on the host directly. Support for this feature in Windows nodes [will now graduate to stable](https://github.com/kubernetes/enhancements/issues/1981) and will be enabled by default.
```

Comment:
```suggestion
Privileged container support allows containers to run with similar access to the host as processes that run on the host directly. Support for this feature in Windows nodes [will now graduate to stable](https://github.com/kubernetes/enhancements/issues/1981).
```

This feature was enabled by default since v1.23 when it entered `beta`.